composer.json

```diff
@@ -103,6 +103,7 @@
         "psr-4": {
             "Symfony\\Bridge\\Doctrine\\": "src/Symfony/Bridge/Doctrine/",
             "Symfony\\Bridge\\Monolog\\": "src/Symfony/Bridge/Monolog/",
+            "Symfony\\Bridge\\PhpUnit\\Compat\\": "src/Symfony/Bridge/PhpUnit/Compat/",
```

Comment:
These classes must be autoloaded by the bridge only. Anything else is wrong.

A failure may be related to cases mixing a downloaded version of the bridge and your patched version here. But I have not seen the failure you are talking about. The only build failures on Travis on the commit before this one are about broken signatures.